src/main/java/io/codelex/arithmetic/practice/Exercise6.java

```diff
@@ -0,0 +1,25 @@
+package io.codelex.arithmetic.practice;
+
+public class Exercise6 {
+    public static void main(String[] args){
```

Comment:
Šis uzdevums neizskatās pabeigts, nosacījumus minēts, ka jāizvad arī woza, bet šeit to neredzu.